extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzRuntimeConfig.java

```diff
@@ -27,4 +27,10 @@
     @ConfigItem
     public boolean forceStart;
 
+    /**
+     * If the quartz scheduler is enabled
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
```

Comment:
There is no need to introduce this config property. Quartz extension is built on top of the Scheduler extension - it's ok to reuse `quarkus.scheduler.enabled`.

Reply:
Ok, I removed the configItem from `QuartzRuntimeConfig`. Now there is only one in `SchedulerRuntimeConfig` called `quarkus.scheduler.enabled`.